tests/common/test_event.py

```diff
@@ -269,13 +268,42 @@ def test_periodic_emits_after_elapsed_delta(self, mock_event):
     @patch('azurelinuxagent.common.event.EventLogger.add_event')
     def test_periodic_forwards_args(self, mock_event):
         event.__event_logger__.reset_periodic()
-
-        event.add_periodic(logger.EVERY_DAY, "FauxEvent")
-        mock_event.assert_called_once_with(
-            "FauxEvent",
-            duration=0, evt_type='', is_internal=False, is_success=True,
-            log_event=True, message='', op=WALAEventOperation.Unknown,
-            version=str(CURRENT_VERSION))
+        event_time = datetime.utcnow().__str__()
+        event.add_periodic(logger.EVERY_DAY, "FauxEvent", op=WALAEventOperation.Log, is_success=True, duration=0,
+                           version=str(CURRENT_VERSION), message="FauxEventMessage", evt_type="", is_internal=False,
+                           log_event=True, force=False, event_time_opcode_name=event_time, event_pid=10, event_tid=
+                           11001001011, task_name="ExtHandler", keyword_name="Yo")
+        mock_event.assert_called_once_with("FauxEvent", op=WALAEventOperation.Log, is_success=True, duration=0,
+                                           version=str(CURRENT_VERSION), message="FauxEventMessage", evt_type="",
+                                           is_internal=False, log_event=True, event_time_opcode_name=event_time,
+                                           event_pid=10, event_tid=11001001011, task_name="ExtHandler", keyword_name=
+                                           "Yo")
+
+    @patch("azurelinuxagent.common.event.datetime")
+    @patch('azurelinuxagent.common.event.EventLogger.add_event')
+    def test_periodic_forwards_args_default_values(self, mock_event, mock_datetime):
+        event.__event_logger__.reset_periodic()
+        event_time = datetime.strptime("2019-01-01 01:30:00", '%Y-%m-%d %H:%M:%S').__str__()
+        mock_datetime.utcnow = mock.Mock(return_value=event_time)
+
+        # Doing the partial to mock the passing of default values of the
```

Comment:
NIT: Incomplete sentence

Reply:
Thanks for catching this. Will fix it now.